### DISCUSSION TITLE
How can i build this xml format ？

### DISCUSSION THREAD

**Discussion by lvlinkeji** [NaturalIntelligence/fast-xml-parser#416] [Q&A]
<voice name="en-US-JennyNeural">
        <prosody rate="+30.00%">
            Welcome to Cognitive Services Text-to-Speech API.
        </prosody>
    </voice>

**Comment by amitguptagwl** [comment]
Do you want it to generate it from js Object? [Here](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/3.XMLBuilder.md#restoring-original-xml) are few examples which may help you.
